Tasks/VsTest/vstest.ts

```diff
@@ -1533,3 +1490,17 @@ function responseContainsNoTests(filePath: string): Q.Promise<boolean> {
         }
     });
 }
+
+function isNullOrWhitespace(input) {
+    if (typeof input === 'undefined' || input == null) {
+        return true;
+    }
+    return input.replace(/\s/g, '').length < 1;
+}
+
+function getTaskMatchOption(): any {
```

Comment:
Added the default match options and added matchBase for consistency with previous implementation of the task